tests/testdata/MutedProblems.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace TestData;
+
+
+use stdClass;
+
+class MutedProblems
+{
+    /** @var stdClass */
+    private $mutedProblems;
```

Comment:
This can also be the class like one in `StubParser::$stubs` and `PHPReflectionParser::$data`

Reply:
It will be done in a separate PR later